src/main/bash/sdkman-init.sh

```diff
@@ -72,17 +74,22 @@ else
 fi
 
 # Source sdkman module scripts.
-for f in $(find "${SDKMAN_DIR}/src" -type f -name 'sdkman-*' -exec basename {} \;); do
-	source "${SDKMAN_DIR}/src/${f}"
+old_IFS="$IFS"
```

Comment:
This block and the following can actually be collapsed into one by using a single `find` command, so I'll add that. Even better would've been to just use shell globbing directly, but dealing with the different ways of enabling `nullglob` in bash and zsh is a hassle.